internal/cmd/cli/cleanup/cleanup.go

```diff
@@ -54,6 +54,7 @@ func ClusterRegistrations(ctx context.Context, client Getter, opts Options) erro
 	// figure out the latest granted registration request per cluster
 	latestGranted := map[types.NamespacedName]metav1.Time{}
 	for _, cr := range crList.Items {
+		cr := cr
```

Comment:
Is this really necessary? I thought Items is `Items []ClusterRegistration`, so not an array of pointers.

Reply:
```go
internal/cmd/cli/cleanup/cleanup.go:63:37: G601: Implicit memory aliasing in for loop. (gosec)
                if cr.Status.Granted && ts.Before(&cr.CreationTimestamp) {
```